### PR TITLE
Code refactoring.

### PR DESCRIPTION
**Pull request by waheedahmed**
VAN-59

### CODE REVIEW

File: src/logistration/RegistrationPage.jsx [hunk 1]
```diff
@@ -31,26 +31,22 @@ class RegistrationPage extends React.Component {
...
-      email: this.state.email,
-      username: this.state.username,
-      password: this.state.password,
-      name: this.state.name,
-      honor_code: true,
-      country: this.state.country,
-    };
-
-    this.props.registerNewUser(payload);
+    if (this.state.formValid) {
```
Comment (zainab-amir) [inline src/logistration/RegistrationPage.jsx:36]: what happens if the form is not valid?

Comment (waheedahmed) [reply src/logistration/RegistrationPage.jsx:36]: @zainab-amir Right now it does nothing but we can show the errors on the unfilled fields.

Comment (zainab-amir) [reply src/logistration/RegistrationPage.jsx:36]: okay

**Review by zainab-amir — APPROVED** [review]

**Review by zainab-amir — APPROVED** [review]
LGTM!

**Pull request merged**